Add canvasColor to theme palette

quanglam2807 opened the pull request:
#1760

shaurya947 commented:
@quanglam2807 you probably just need to update the last unit test.

Thanks for this. Could you make sure that you have updated *all* source files so that any kind of background property now uses the `canvasColor` instead of `alternateTextColor`?

quanglam2807 commented:
@shaurya947 Actually, I changed the alternateTextColor to `Colors.white` so it failed the test and I have set it back to the original value so it's working now.
Also, I am pretty sure that I have changed all the background values of `src` and `docs` to `canvasColor`. No need to say thanks! Your work is amazing and I just want to help a little bit. :+1:
